flows/nodes/queryalert.md

```diff
@@ -0,0 +1,40 @@
+# Query Log Ingest Node
+
+The Query Log Ingest node is an advanced node which generates *alert entries* based on the results of a previous Gravwell query, then ingests those alerts to specified tag.
+
+## Configuration
+
+* `Destination Tag`, required: the tag into which the alerts should be ingested. The default of `_alert` is a good choice.
```

Comment:
Add a note that the `_alert` tag is the Gravwell standard tag for alerts, choosing an alternate tag means that your alerts may not show up in dashboards and queries deployed by the Gravwell kit.